### ISSUE TITLE
Book covers will not use local versions.

### ISSUE THREAD

**Issue by brandontravis** [seanap/Plex-Audiobook-Guide#13]
I have everything working properly, but no matter what I do I cannot get Plex to use the local artwork instead of the Audible artwork. When using MP3Tag to import, I made sure that the artwork I wanted was the ONLY cover image listed. Plex properly sees this artwork when I change the agent to Plex Music. Am I doing something wrong? Is this working as intended?

Any help is much appreciated.

**Comment by seanap** [comment]
Something doesn't sound right, the covers should be pulled in from the file. Are you using my version of the Plex Audiobook Agent, or macr0dev's?

In my Audiobook Plex Library settings, I have "Audiobook" selected as the Agent, and I also set Album Art = "Local Files Only"

My version of the audiobook agent specifically does NOT set the cover, because I am assuming that the actual files will have your preferred cover embedded.  I do this because sometimes I don't like the Audible cover, and I don't want to go through the work of tagging the files with the correct cover only to have the agent scrape the audible cover.

**Comment by brandontravis** [comment]
I am using your copy, and have it set the exact same as you. Here is an example of a book in MP3Tag (using my cover) compared to the book in Plex using your agent. The cover shown here is the one from Audible. 
![mp3tag](https://user-images.githubusercontent.com/5884247/105551338-98b6c580-5cd0-11eb-91b5-640c1707c9bb.PNG)
![plex](https://user-images.githubusercontent.com/5884247/105551340-994f5c00-5cd0-11eb-8852-7c06f8e8754a.PNG)

Also confirmed that the cover.jpg image that gets output by your MP3tag actions match the cover seen in the MP3Tag screenshot.

For what its worth, these are m4b files. Not sure if there would be any difference with mp3. 

I am completely at a loss with this one. Luckily I am using Audibles covers a significant portion of the time, so I can work around this issue, but would love to figure out what I'm doing wrong. Anyways, thanks for the help.

**Comment by seanap** [comment]
Now I'm very confused as well, lol

My version of 11.22.63 had cover1, I duplicated the folder and edited the tag of the copy to cover2. Plex imported the new copy but changed both albums (original and copy) to cover2.  I unmatched the original album and now it has cover3...  I have no clue where it's pulling cover3 from.  That said, my copy matched to the custom cover2 as expected.

The agent must be doing something it's not supposed to.  This is very weird.
